server/controllers/groups/lib/membership_actions.coffee

```diff
@@ -1,12 +1,20 @@
 CONFIG = require 'config'
 __ = CONFIG.universalPath
 { membershipActionsList } = __.require 'models', 'group'
+Group = __.require 'models', 'group'
+radio = __.require 'lib', 'radio'
+initMembershipUpdateHooks = require './membership_update_hooks'
```

Comment:
shall this `membershipUpdate` be part of the PR? It seems to me that it is not that much related to group invitation. Small commit + small PR keep the review doable. cuz now its a bit confusing to me.

Reply:
it's part of the PR in the sense that I needed this refactor 8fbb973 to understand how it works: hopefully, when seen in the commit view, and if we keep committed on using the term *refactor*  only for code changes that don't affect the interface or behavior, it shouln't be that confusing